code/analyze/ac/stats/de_StatsFreqPreferences.m

```diff
@@ -51,6 +50,12 @@
           if si == 1
             stats.(prop).mean = cell(n_sigmas, 1);
             stats.(prop).std = cell(n_sigmas, 1);
+            if strcmp(prop, 'freq') % This code is used for sigma vs xover computations
```

Comment:
When you say "everything from line 52", do you mean line 52 to line 57? Line 52 to line 61? Not 100% sure I follow. I can easily move lines 54-57 (the code I added) to before the loop. However lines 52-53 will require a for loop (over property names, i.e. `prop`), and lines 60-61 will require a for loop over `prop` as well as over `n_sigmas.`

Is that what you had in mind?